### ISSUE TITLE
Add confusion matrix plotter for MSE

### ISSUE THREAD

**Issue by Lyubava** [Samsung/veles.znicz#4]
Confusion matrix plotter works only for softmax loss function
